modules/op-starting-pipelines-from-topology-view.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="op-starting-pipelines_from_topology_view{context}"]
+= Starting pipelines from Topology view
+
+For pipelines created using the *From Git* option, you can use the *Topology* view to interact with pipelines after you start them:
+
+[NOTE]
+====
+To see the pipelines created using the *Pipeline Builder* in the *Topology* view, customize the pipeline labels to link the pipeline with the application workload.
+====
+.Procedure
+
+. Click *Topology* in the left navigation panel.
+. Click the application to display *Pipeline Runs* in the side panel.
+. In *Pipeline Runs*, click *Start Last Run* to start a new pipeline run with the same parameters and resources as the previous one. This option is disabled if a pipeline run has not been initiated. You can also start a pipeline run when you create it.
++
+.Pipelines in Topology view
+image::op_pipeline_topology.png[Pipelines in Topology view]
++
+. In the *Topology* page, hover to the left of the application to see the status of its pipeline run. When the pipeline is added, the bottom left decorator indicates that there is an associated pipeline.
++
+[NOTE]
+====
+The side panel of the application node in the *Topology* page displays a *Log Snippet* when a pipeline run fails on a specific task run. You can view the *Log Snippet* in the *Pipeline Runs* section, under the *Resources* tab. *Log Snippet* provides a general error message and a snippet of the log. A link to the *Logs* section provides quick access to the details about the failed run.
+====
+. If a Pipeline run does not start automatically, the side panel displays a message that the pipeline cannot be automatically started. On the other hand, if a Pipeline is created but the user has not started the pipeline, its status is not started. When the user clicks the not started status icon, the start dialog box opens in the *Topology* view.
```

Comment:
This is descriptive information, not a step a user can perform. Please remove the numbering. The step on line 24 also seems informational. 
Consider splitting everything from line 21 and down into a separate conceptual topic and present this information as a series of bullet points. What do you think?

Reply:
Following the Mod Doc standard for content types is an important OKR. I think fixing this issue is important for us to comply with the Mod Docs standard.

Reply:
@rolfedh Can you please have a look and see if all looks correct?